tests/conftest.py

```diff
@@ -2899,3 +2908,126 @@ def fixture_demultiplexed_flow_cells_tmp_directory(tmp_path) -> Path:
     tmp_dir = Path(tmp_path, "tmp_run_dir")
 
     return Path(shutil.copytree(original_dir, tmp_dir))
+
+
+## Taxprofiler fixtures
+
+
+@pytest.fixture
+def taxprofiler_config(taxprofiler_dir: Path, taxprofiler_case_id: str) -> None:
+    """Create samplesheet.csv file for testing"""
+    Path.mkdir(Path(taxprofiler_dir, taxprofiler_case_id), parents=True, exist_ok=True)
+    Path(taxprofiler_dir, taxprofiler_case_id, f"{taxprofiler_case_id}_samplesheet.csv").touch(
```

Comment:
```suggestion
    Path(taxprofiler_dir, taxprofiler_case_id, f"{taxprofiler_case_id}_sample_sheet.csv").touch(
```
Use FileExtension constant instead of ".csv"

Reply:
Done.